docs/ecosystem/officials/zio-mock.md

```diff
@@ -916,7 +927,6 @@ test("effectful expectation") {
         Random
           .nextUUID
           .map(id => User(id.toString, s"name-$n"))
-          .provideLayer(Random.live)
```

Comment:
Please check this gist when reviewing this change.
https://gist.github.com/JoaquinIglesiasTurina/10d16eed4ab774171525673f3087d8e8

Reply:
👍